### PR TITLE
Minor updates to product.yml and variable usage

### PR DESCRIPTION
**Pull request by qahmed1998**

### CODE REVIEW

**Comment by qahmed1998** [comment]
@miq-bot assign @Fryguy

**Comment by qahmed1998** [comment]
@miq-bot add_labels radjabov/yes

**Comment by Fryguy** [comment]
@agrare Please review.  I think this is ok, but I don't know the details.

File: managing_providers/_topics/automation_management_providers.md [hunk 1]
```diff
@@ -381,14 +381,14 @@ If you want to use the Embedded Terraform feature in {{ site.data.product.title_
...
-runuser --login manageiq --command 'podman --root=/var/www/miq/vmdb/data/containers/storage image load --input /tmp/<OpenTofu_image>'
+{{ site.data.product.import_opentofu }}
 ```
 
 Where `OpenTofu_image` is the name of your OpenTofu image.
 
 You also need to set the docker image name in advanced settings before enabling the server role. Navigate to the **Settings** > **Application Settings** in {{ site.data.product.title_short }} UI and set the value for `workers/worker_base/opentofu_worker/container_image` field.
 
-An example value of this field is `container_image: docker.io/manageiq/opentofu-runner:latest`.
+An example value of this field is `container_image:` `{{ site.data.product.container_image }}`.
```
Comment (agrare) [inline managing_providers/_topics/automation_management_providers.md:391]: Is there an extra couple of ` in there?
```suggestion
An example value of this field is `container_image: {{ site.data.product.container_image }}`.
```

Comment (qahmed1998) [reply managing_providers/_topics/automation_management_providers.md:391]: Oh true, you are correct.

**Review by agrare — APPROVED** [review]

**Comment by Fryguy** [comment]
Backported to `radjabov` in commit fea75352865e88389a7b37acb1842f68ec18f354.

```text
commit fea75352865e88389a7b37acb1842f68ec18f354
Author: Adam Grare <adam@grare.com>
Date:   Wed Jul 10 15:38:22 2024 -0400

    Merge pull request #1796 from qahmed1998/update-terraform
    
    Minor updates to product.yml and variable usage
    
    (cherry picked from commit 269efa281412f397e1693c84f2ea836daf28266e)
```

**Pull request merged**